gum/backend-libdwarf/gumsymbolutil-libdwarf.c

```diff
@@ -344,6 +356,56 @@ gum_find_function (const gchar * name)
   return address;
 }
 
+static gboolean
+gum_find_nearest_symbol_by_address (gpointer address,
+                                    GumNearestSymbolDetails * nearest)
+{
+  GHashTable * table;
+  gchar * name;
+  GList * keys, * cur;
+  gpointer current_address, closest_address;
```

Comment:
Perhaps we should name it `nearest_address`, so we only use one term?

Reply:
Sounds good.